pytorch_lightning/plugins/io/hpu_io_plugin.py

```diff
@@ -0,0 +1,73 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
`hpu_io_plugin` is redundant sinde this is already under the `io/` directory. `hpu_plugin` should be fine